src/components/Header/index.js

```diff
@@ -25,6 +23,8 @@ class Header extends Component {
   toggleMenu = () => this.setState(state => ({ menuOpen: !state.menuOpen }))
 
   render() {
+    const { location } = this.props
```

Comment:
If we want to force this value to default to an empty string, we could put this here?
```
const { location = '' } = this.props
```

Reply:
That is what the defaultProps does 🙂

Reply:
Yeah. Only mentioned cos of my misunderstanding above :sweat_smile: